content/workload-reduction-toolkit/address-workload-issues/behaviour-management/review-and-streamline-behaviour-management.md

```diff
@@ -95,3 +95,28 @@ Provide a supportive environment for open and honest professional dialogue.
 - [Policy Exchange publication: It just grinds you down](https://policyexchange.org.uk/publication/it-just-grinds-you-down)
 
 - [Making data work: Teacher workload advisory group report](https://www.gov.uk/government/publications/teacher-workload-advisory-group-report-and-government-response)
+
+<div class="dfe-width-container govuk-grid-row">
```

Comment:
This width container is making the box unaligned with the content above it on smaller screens. We can just get rid of this wrapping div. I'm also removing all these in my mobile layout PR.

```suggestion
```

**How it looks currently:**

<img width="589" alt="Screenshot 2024-03-21 at 11 47 01" src="https://github.com/DFE-Digital/improve-workload-and-wellbeing-for-school-staff/assets/18436946/9a34a1c1-9d10-4aee-9437-e92e448a9f1b">